packages/edit-site/src/components/routes/use-title.js

```diff
@@ -47,14 +47,17 @@ export default function useTitle( title ) {
 			document.title = formattedTitle;
 
 			// Announce title on route change for screen readers.
+			if ( ! accessibleTitle ) {
+				return;
+			}
 			speak(
 				sprintf(
 					/* translators: The page title that is currently displaying. */
 					__( 'Now displaying: %s' ),
```

Comment:
Is the prefix "Now displaying" necessary? Could that be shortened? E.g., "Showing: %s". I feel like we don't need to indicate that it's happening now; that should be obvious from context, so dropping this from four syllables to two would be helpful. I'm not even 100% convinced the prefix is required at all; page navigation in a standard context doesn't prefix the `title` element.

Reply:
@joedolson Agreed. Making this change.

Reply:
Done.